build.sbt

```diff
@@ -23,12 +23,26 @@ lazy val root = (project in file(".")).enablePlugins(
 
 scalaVersion := "2.11.6"
 
+val cxfV = "2.7.5"
+val cxfUtilsV = "2.7.0"
+val exactTargetDeps = Seq(
+  "com.exacttarget" % "fuelsdk" % "1.0.3" from "https://github.com/salesforcefuel/FuelSDK-Java/releases/download/v1.0.3/fuel-java-1.0.3.jar",
```

Comment:
They don't even upload to Maven Central?! Gah! Come on! https://github.com/salesforcefuel/FuelSDK-Java/issues/13